Basic-Car-Maintenance/Shared/Dashboard/Views/ExportOptionsView.swift

```diff
@@ -41,22 +49,39 @@ struct ExportOptionsView: View {
             .padding(.horizontal)
             .toolbar {
                 ToolbarItem(placement: .topBarTrailing) {
-                    Button("Export") { 
+                    Menu(content: {
+                        Picker("Export", selection: $selectedOption) {
+                            ForEach(ExportOption.allCases, id: \.self) { option in
```

Comment:
```suggestion
                            ForEach(ExportOption.allCases) { option in
```

The `id: \.self` isn't needed because `ExportOption` conforms to `Identifiable`

Also for the Menu I use multiple trailing closure syntax, as a personal preference